account_default_draft_move/account.py

```diff
@@ -55,3 +55,14 @@ def button_cancel(self):
                              'SET state=%s '
                              'WHERE id IN %s', ('draft', tuple(self.ids)))
         return True
+
+    @api.multi
+    def _is_update_posted(self):
+        ir_module = self.env['ir.module.module']
+        can_cancel = ir_module.search([('name', '=', 'account_cancel'),
+                                       ('state', '=', 'installed')])
+        for move in self:
+            self.update_posted = can_cancel and move.journal_id.update_posted
```

Comment:
:bug: 
it should be: `move.update_posted = can_cancel and move.journal_id.update_posted`